scripts/renderers.py

```diff
@@ -99,14 +123,14 @@ def sorting_header(key):
         output.extend(header)
         return output
 
-    def render_csv_report(self, envs_ida_toggle_data, file_path="report.csv", toggle_types=None, header=None):
+    def summarize_data(self, toggles_data):
         """
-        takes data, processes it, and outputs it in csv form
+        Returns only subset containing the essential information
         """
-        toggles_data = self.transform_toggle_data_for_csv(envs_ida_toggle_data)
-        data_to_render = self.filter_and_sort_toggles(toggles_data, toggle_types)
-        header = self.get_sorted_headers_from_toggles(data_to_render, header)
-        self.write_csv(file_path, data_to_render, header)
+        data_to_render = []
+        for ida_name, ida in toggles_data.items():
+            data_to_render.extend(ida.get_toggles_data_summary())
+        return data_to_render
```

Comment:
```
return [ida.get_toggles_data_summary() for ida in toggles_data.values()]
```

Reply:
I think you are missing the extend in the lines above. get_toggles_data_summary() returns a list full of toggle info dicts

Reply:
Right! I mistook `extend` with `join`. So we can write instead:

```
return sum(list(ida.get_toggles_data_summary()) for ida in toggles_data.values())
```

In any case, we should not iterate over the dict `items` but on its `values`, as the keys are not used.

Reply:
I agree with change to values instead on items, but don't like putting this in one line. That seems less readable to me

Reply:
Sure :+1: